torch/_inductor/autoheuristic/autoheuristic_utils.py

```diff
@@ -219,10 +221,14 @@ def pow2_op(data: Any, dim: str, exponent: int) -> bool:
     return data[dim] == 2**exponent
 
 
-def mixed_mm_operations() -> List[AHOperation]:
+def mm_operations() -> List[AHOperation]:
     mult_dims_ops = get_mult_dims_ops()
     arith_intensity_op = AHOperation("arith_intensity", get_arith_intensity)
-    return mult_dims_ops + [arith_intensity_op] + between_ops()
+    return mult_dims_ops + [arith_intensity_op]
```

Comment:
Maybe useful to add memory bandwidth in the future

Reply:
Can you explain? Currently, the learned heuristics are GPU specific, so I assume memory bandwidth to always be the same? Or do you mean, if we wanted to learn one heuristic in total instead of one per GPU, we would want to add some GPU information as features (bandwidth, shared memory etc.)?